pkg/api/plugins.go

```diff
@@ -258,6 +259,14 @@ func (hs *HTTPServer) UpdatePluginSetting(c *contextmodel.ReqContext) response.R
 		return response.Error(http.StatusNotFound, "Plugin not installed", nil)
 	}
 
+	p, found := hs.pluginStore.Plugin(c.Req.Context(), pluginID)
+	if !found {
+		return response.Error(http.StatusNotFound, "Plugin not found", nil)
+	}
```

Comment:
There's already a call to `hs.pluginStore.Plugin` right before this one, but the first returned argument is discarded. Can we re-use that one instead?

Reply:
good catch, I missed that one!